### PR TITLE
test: added Token Claim Airdrop Test

### LINKED ISSUE #14959: (CLAIM_59) TokenClaimAirdrop transaction with a receiver - a hollow account that signs it with no HBAR fails

### PR DESCRIPTION
**Pull request by iwsimon**
**Description**:

(CLAIM_59) TokenClaimAirdrop transaction with a receiver - a hollow account that signs it with no HBAR fails

**Related issue(s)**:

Fixes #14959

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by david-bakin-sl — APPROVED** [review]
LGTM!  

By the way, would you mind going up to L243 and changing the misspelling "Calim" -> "Claim"?

**Comment by iwsimon** [comment]
> "Calim" -> "Claim"

Updated it

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java [hunk 1]
```diff
@@ -736,6 +745,26 @@ final Stream<DynamicTest> tokenClaimAirdropDisabledFail() {
...
+        final var hollowAcnt = "hollowAcnt";
+        return hapiTest(
+                cryptoCreate(OWNER).balance(ONE_HUNDRED_HBARS),
+                createFT(FUNGIBLE_TOKEN_1, OWNER, 1000L),
+                newKeyNamed(hollowAcnt).shape(SECP_256K1_SHAPE),
+                createHollowAccountByFunToken(hollowAcnt, FUNGIBLE_TOKEN_1, OWNER),
+                tokenAirdrop(moving(10, FUNGIBLE_TOKEN_1).between(OWNER, hollowAcnt))
+                        .payingWith(OWNER),
+                getAliasedAccountInfo(hollowAcnt)
+                        .has(accountWith().hasEmptyKey().noAlias()),
```
Comment (ibankov) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java:760]: you can replace this with getAliasedAccountInfo(hollowAcnt).isHollow()

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java [hunk 2]
```diff
@@ -747,4 +776,22 @@ private HapiTokenAirdrop airdropFT(String tokenName, String sender, String recei
         return tokenAirdrop(moving(amountToMove, tokenName).between(sender, receiver))
                 .payingWith(sender);
     }
+
+    private SpecOperation createHollowAccountByFunToken(String hollowAcnt, String token, String owner) {
```
Comment (ibankov) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java:780]: There is a createHollow() in UtilVerbs.java

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java (hunk 1 above)
Comment (iwsimon) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java:760]: Thanks @ibankov. I just checked .isHollow(), only checked `assertEquals(IMMUTABILITY_SENTINEL_KEY, key);`. Here I also check .noAlias().

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java (hunk 2 above)
Comment (ibankov) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java:780]: You can use the 
```
public static SpecOperation createHollow(
            final int n,
            @NonNull final IntFunction<String> nameFn,
            @NonNull final Function<Address, HapiCryptoTransfer> creationFn)
```
method and pass tokenTransfer for creationFn

Comment (iwsimon) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java:780]: That method creates more overhead than needed in this situation.

**Pull request merged**